changelogs/fragments/169.yml

```
@@ -0,0 +1,3 @@
+major_changes:
+  - Add mysql_sha2_password encryption for Proxysql users definitions, this improve security and also solve the issue about deprecation of mysql_native_password.
+    Until version 2.0 of community.proxysql, by default mysql_native_password remain the default choice.
```

```suggestion
minor_changes:
  - proxysql_mysql_users - Creating users with the ``caching_sha2_password`` plugin (https://github.com/ansible-collections/community.proxysql/pull/164).
```
this will appear in CHANGELOG.rst automatically
also it's not a role, it's a module